doc/03-Configuration.md

```diff
@@ -35,4 +61,15 @@ database:
   password: icingadb
 redis:
   address: redis:6380
+cleanup:
+  history:
+    acknowledgement: 10
```

Comment:
Suggestion: I think the configuration would be way more self-documenting if this was structured more like the following:
```yaml
history:
  retention-days:
    acknowledgement: 10
```

Also, it is nice to be able to have this fine-grained configuration, but I think for most users it would be sufficient to have a single "retain 100 days of history" option. So maybe have one option that's used as a default for all these detailed options.